ballerina/tests/tests.bal

```diff
@@ -0,0 +1,347 @@
+// Copyright (c) 2025, WSO2 LLC. (http://www.wso2.com).
+//
+// WSO2 LLC. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+import ballerina/http;
+import ballerina/test;
+import ballerina/time;
+
+configurable string serviceUrl = isLiveServer ? "https://api-m.sandbox.paypal.com/v1/billing" : "http://localhost:9090/v1/billing";
+configurable string clientId = ?;
+configurable string clientSecret = ?;
+string testProductId = "";
+string testPlanId = "";
+string testSubscriptionId = "";
+configurable string testActivatedSubscriptionId = ?;
+string testActivatedSubscriptionPlanId = "";
```

Comment:
Keep a separation between the `configurable` variables and the other global variables.

```suggestion
configurable string serviceUrl = isLiveServer ? "https://api-m.sandbox.paypal.com/v1/billing" : "http://localhost:9090/v1/billing";
configurable string clientId = ?;
configurable string clientSecret = ?;
configurable string testActivatedSubscriptionId = ?;

string testProductId = "";
string testPlanId = "";
string testSubscriptionId = "";
string testActivatedSubscriptionPlanId = "";
```